recipes/torchtime/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "torchtime" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchtime-{{ version }}.tar.gz
+  sha256: 45ce7d95fa9974f0c6edb967cf08546f68b00e49d0f997a6a69281900f3de886
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.8,<3.10
+    - pip
+    - poetry >=1
+  run:
+    - python >=3.8,<3.10
+    - sktime
+    - requests
+    - scikit-learn
+    - tqdm
+    - pytorch  # pytorch is globally pinned
```

Comment:
You should include the constraints on the runtime dependencies except pytorch

https://github.com/philipdarke/torchtime/blob/7feb206557905282aba4b7bcbf5ab939794b7972/pyproject.toml#L13